bitrise.yml

```diff
@@ -174,18 +170,25 @@ workflows:
         inputs:
         - repository_url: $SAMPLE_APP_URL
         - branch: $BRANCH
-        - clone_into_dir: "."
-    - certificate-and-profile-installer: {}
+        - clone_into_dir: .
+    - certificate-and-profile-installer: { }
     - script:
         inputs:
         - content: |-
-            #!/usr/bin/env bash
             CURRENT_BUILD_NUMBER=${BITRISE_BUILD_NUMBER}
+
+            if [[ -z "$E2E_ORIG_BUILD_NUM" ]]; then
```

Comment:
Since we're uploading this project multiple times in the same workflow to Appstore Connect the current build number kept getting added to the newly generated build number. This resulted in the following example build versions in Appstore Connect:
```
2210292233420769

4420584468870769

6630876706910769

8841168946420769
```

We decided that the generated timestamp is enough for uniqueness and just add the build number at the end:
```
2211080410320798

2211080412560798

2211080415140798

2211080417010798
```

This may have been the intent of the original code, but was resulting in build failures as the last build version from the previous build (8841168946420769) was larger than the first build version from the current build (2211080410320798).

I'm not sure what changed from before when it was working fine, but here's a build where it was also failing on `master`: https://app.bitrise.io/build/d2265161-d280-415c-b2f1-2d66455dd28c